homeassistant/components/tesla_fleet/coordinator.py

```diff
@@ -182,6 +182,80 @@ async def _async_update_data(self) -> dict[str, Any]:
         return data
 
 
+class TeslaFleetEnergySitePastCoordinator(DataUpdateCoordinator[dict[str, Any]]):
+    """Class to manage fetching energy site past import and export from the TeslaFleet API."""
+
+    updated_once: bool
+
+    def __init__(self, hass: HomeAssistant, api: EnergySpecific) -> None:
+        """Initialize TeslaFleet Energy Site Past coordinator."""
+        super().__init__(
+            hass,
+            LOGGER,
+            name="Tesla Fleet Energy Site Past",
+            update_interval=timedelta(seconds=10),
+        )
+        self.api = api
+        self.data = {}
+        self.updated_once = False
+
+    async def _async_update_data(self) -> dict[str, Any]:
+        """Update energy site past data using TeslaFleet API."""
+
+        self.update_interval = ENERGY_INTERVAL
+        n = datetime.now(UTC).isoformat()
+        try:
+            data = (await self.api.energy_history("energy", n, n, "day", "UTC"))[
```

Comment:
The parameters for this have changed. I recommend just pulling 1 day in the installations timezone like this.

```suggestion
            data = (await self.api.energy_history("day"))[
```